server/game/cards/02_SHD/units/TarffulKashyyykChieftain.ts

```diff
@@ -0,0 +1,36 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { DamageType, RelativePlayer, Trait, WildcardCardType, ZoneName } from '../../../core/Constants';
+
+export default class TarffulKashyyykChieftain extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '7022736145',
+            internalName: 'tarfful#kashyyyk-chieftain'
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addTriggeredAbility({
+            title: 'That unit deals that much damage to an enemy ground unit',
+            when: {
+                onDamageDealt: (event, context) =>
+                    event.type === DamageType.Combat &&
+                    event.card.hasSomeTrait(Trait.Wookiee) &&
+                    event.isResolved &&
+                    event.card.canBeInPlay() && event.card.isInPlay() && !event.card.pendingDefeat &&
+                    event.card.controller === context.source.controller
+            },
+            targetResolver: {
+                controller: RelativePlayer.Opponent,
+                cardTypeFilter: WildcardCardType.Unit,
+                zoneFilter: ZoneName.GroundArena,
+                immediateEffect: AbilityHelper.immediateEffects.damage((context) => ({
+                    amount: context.event.amount
+                }))
```

Comment:
This will use Tarfful as source of the damage instead of the unit that was attacked, I'm not sure if it's important.

Reply:
Unfortunately I think it probably is, it's not hard to imagine that there could be situations where it's important which specific unit is dealing damage

Reply:
That's what I thought as well, but there are implemented cards with similar wording (Strike True for example) that don't do anything special. I think those will need to be updated as well!

Reply:
Yeah correct, they will :/ it's on the engine work items list I believe

Reply:
I think I've fixed all the existing cases, Command will be another one but it's implemented yet.